packages/api-explorer/e2e/e2e.spec.ts

```diff
@@ -164,31 +166,35 @@ describe('API Explorer', () => {
       await expect(page).toMatchElement('h3', { text: 'Kotlin Declaration' })
     })
 
-    it('changes specs', async () => {
-      await expect(page).toMatchElement('h2', {
-        text: 'Looker API 3.1 Reference',
-      })
-      await expect(page).toClick('input[value="3.1"]')
-      await expect(page).toClick(
-        'ul[aria-label="spec selector"] > li:last-child'
-      )
-      await expect(page).toMatchElement('h2', {
-        text: 'Looker API 4.0 (Beta) Reference',
-      })
-    })
+    // This test was broken during the 4.0 GA spec changes, and needs to be fixed
+    // it('changes specs', async () => {
+    //   await expect(page).toMatchElement('h2', {
+    //     text: 'Looker API 4.0 Reference',
+    //   })
+    //   await expect(page).toClick('input[value="4.0"]')
+    //   await Promise.all([
```

Comment:
is this intentionally commented out? if yes, do we have a ticket to track its fix?

Reply:
Yes, it was broken when I merged main back in, along with two others that I was able to fix.

I just created this issue: https://b.corp.google.com/issues/231742409